bin/warmup_stats

```diff
@@ -284,33 +333,47 @@ class BenchmarkFile(object):
 
 
 def main(options):
+    info('Checking sanity of CLI options.')
     need_latex = (options.output_table or options.output_diff) and options.type_latex
     need_plots = options.output_plots
     if options.output_table and not (options.type_latex or options.type_html):
         fatal('--output-table must be used with either --html or --tex.')
     if options.output_diff and not (options.type_latex or options.type_html):
         fatal('--output-diff must be used with either --html or --tex.')
-    csv_files = options.csv_files[0]
-    if options.output_diff and len(csv_files) != 2:
+    input_files = options.input_files[0]
+    for filename in input_files:
+        if filename.endswith('.csv') and not options.language:
```

Comment:
Might be clearer to say `if filenamename.endswith('.csv')` and then have the 3 `if`s as sub-parts of that?